src/Validation/ItValidation.php

```diff
@@ -97,6 +139,6 @@ public static function cf($check)
      */
     public static function personId($check)
     {
-        throw new NotImplementedException(__d('localized', '%s Not implemented yet.'));
+        return cf($check);
```

Comment:
cf()?

Reply:
cf is the function for Codice Fiscale, in Italy is the Person ID.
https://en.wikipedia.org/wiki/National_identification_number#Italy

Reply:
Is this always available in PHP?

Reply:
Line 58 in src/Validation/ItValidation.php

Reply:
Just calling cf() will not work. Please add a test case here.
You need to call it with late static binding: static::cf() etc

Reply:
Sorry. You're right